source/docs/api-reference/api-usage/control-requests.rst

```diff
@@ -86,6 +86,41 @@ Control requests also supports modification using method chaining. This can be u
          // mutate request with output of 10 amps and max duty cycle 0.5
          m_motor.SetControl(motorRequest.WithOutputAmps(10_A).WithMaxDutyCycle(0.5));
 
+Limit Switches
+--------------
```

Comment:
This section doesn't belong here. Really what we want is:
- In hardware section, mention support for limit switches, what pins are what, blink codes, etc., and link to the software section
- New "Limits" page in software section that covers supply/stator current limits, hardware limits, software limits, and document things in terms of configs, not the status signal or control request.